doc/plutus-core-spec/cardano/versions.tex

```diff
@@ -106,6 +106,7 @@ \section{Ledger languages}
         \LL{PlutusV2} & 7.0 & Batch 2 \\
         \LL{PlutusV2} & 8.0 & Batch 3 \\
         \LL{PlutusV3} & 9.0 & Batch 4 \\
+        \LL{PlutusV3} & 10.0 & Batch 5 \\
```

Comment:
This is for future work: we'll need to rework or remove this table, since it would no longer make sense when we start enabling existing builtins for Plutus V1 and V2.

Reply:
See [this issue](https://github.com/IntersectMBO/plutus-private/issues/1119).